Elle624/ivolunteer
# Feature/welcome page

#### What's this PR do?
Add a welcome page showing app title, tagline and redirect button to a homepage
Add link to a button 
Add animation to the elements on the page to improve UX
Add a home page button image 

#### What does it fix?
N/A

#### Where should the reviewer start?
App.test.js line 22
PostingView.test.js line 2
PostingView/index.js line 79
WelcomePage.scss line 1
WelcomePage.test.js line 1
WelcomePage/index.js line 1

#### How should this be manually tested?
By running `npm start' and 'npm test' all test should pass and the landing page of the website should start with WelcomePage component

#### Any background context you want to provide?


#### Screenshots (if appropriate):
<img width="1363" alt="Screen Shot 2021-01-11 at 11 07 45 PM" src="https://user-images.githubusercontent.com/66269306/104276393-fc83f780-5461-11eb-9ca9-26248721f238.png">


#### Questions


#### Related issues
